tasks/Task.cpp

```diff
@@ -42,8 +57,159 @@ void Task::errorHook()
 void Task::stopHook()
 {
     TaskBase::stopHook();
+    ouster::sensor::sensor_config config;
+    config.operating_mode = ouster::sensor::OperatingMode::OPERATING_STANDBY;
+    if (!sensor::set_config(m_sensor_hostname, config, 0)) {
+        LOG_ERROR_S << "Failed to configure Lidar!" << std::endl;
+    }
+    m_handle.reset();
```

Comment:
Acho melhor initializar o client no configureHook e guardar ele.

Reply:
Since you moved the m_handle initialization to the configureHook, you should be reseting the pointer in the cleanupHook

Reply:
done